shows.js

```diff
@@ -0,0 +1,60 @@
+//Constructor for building show objects
+var shows = [];
+function Show(title, servicesArray) {
+  this.servicesArray = servicesArray;
+  this.show_title = title;
+  this.prod_year = null;
+  this.rating = null;
+  shows.push(this);
+}
+
+var himym = new Show('How I Met Your Mother', ['Netflix', 'Hulu']);
```

Comment:
Given that you're storing all of these shows in the `shows` array, you don't need to explicitly assign each of them to variables.